### PR TITLE
Create Code of Conduct page

### PR DESCRIPTION
**Pull request by kylefarmer85**
[ticket](https://trello.com/c/CZq511FN/163-voma-code-of-conduct-agreement-page)

I referenced the existing components for styling. It doesn't look exactly like the mockup on Mira, but it matches the other components. I was thinking there should be a more simple solution to disabling the next button in the CodeOfConduct component, but this was the best I could come up with.

### CODE REVIEW

File: src/components/CodeOfConduct.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      <hr />
+      <br />
+      <p>Code for Chicago&apos;s Code of Conduct</p>
+      <p>
+        Adapted from the{' '}
+        <a href="https://github.com/codeforamerica/codeofconduct">CfA Code of Conduct</a>. We are an
+        official brigade. <br />
+        The Code for Chicago community expects that Code for Chicago events, and digital forums:
+      </p>
+      <p>
```
Comment (daleinen7) [inline src/components/CodeOfConduct.jsx:22]: Might make more sense to turn this into an `<ol>`

File: src/components/CodeOfConduct.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        1. Are a safe and respectful environment for all participants. <br />
```
Comment (daleinen7) [inline src/components/CodeOfConduct.jsx:23]: and these as `<li>`

File: src/components/CodeOfConduct.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        participate in Code for Chicago network activities, events, and digital forums.
+      </p>
+      <hr />
+      <p className="instructions">Have you read this?</p>
+      <label className="skillLabel" htmlFor="COC">
+        <input type="radio" id="COC" onChange={() => setAccepted(true)} />I understand and I accept
+        this Privacy Policy.
+      </label>
+      <nav>
+        <button onClick={() => setRegisterStep(1)} type="button">
```
Comment (daleinen7) [inline src/components/CodeOfConduct.jsx:56]: This goes back to the beginning and should probably go back to the skills fieldset. I think setting `setRegisterStep(2)` should do the trick.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Looks good. Just a few small changes.

File: src/components/CodeOfConduct.jsx (hunk 3 above)
Comment (kylefarmer85) [reply src/components/CodeOfConduct.jsx:56]: Good catch. The formatting with the `ol` looks slightly odd, maybe we should modify the `ol` in the styled fieldset?

**Review by daleinen7 — CHANGES_REQUESTED** [review]
I see what you're saying about the default style. Yeah, should be able to fix all that in the StyledFieldset component. Otherwise looks good to me!

**Review by daleinen7 — APPROVED** [review]
Looks good to me!

**Pull request merged**